demos/link-prediction/gcn/cora-gcn-links-example.ipynb

```diff
@@ -35,7 +35,9 @@
     "from tensorflow import keras\n",
```

Comment:
<p>What are these <code>['...']</code>  s that have been introduced in the output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/816/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
well spotted, these are tensorflow warnings that aren't completely removed by the `format_notebooks.py` script; eg before stripping they were:
```
WARNING:tensorflow:sample_weight modes were coerced from
  ...
    to  
  ['...']
WARNING:tensorflow:sample_weight modes were coerced from
  ...
    to  
  ['...']
```

Reply:
Filed #826 as this issue is bigger than just this notebook